scripts/inspect_replay_and_td.py

```
@@ -0,0 +1,302 @@
+        state = dataset_features.iloc[i].to_numpy(dtype=float)
+        next_state = dataset_features.iloc[i + 1].to_numpy(dtype=float)
+        reward = float(close_prices[i + 1] - close_prices[i])
+        action = random.randint(0, 2)
+        done = i == max_idx - 1
+        replay.append((state, action, reward, next_state, done))
+    return replay
+
+
+def load_replay(replay_path: Path) -> List[Tuple]:
+    with open(replay_path, "rb") as fh:
+        return pickle.load(fh)
```

Ruff (0.14.5)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)